libvast/src/plugin.cpp

```diff
@@ -54,14 +54,13 @@ detail::stable_set<std::filesystem::path>
 get_plugin_dirs(const caf::actor_system_config& cfg) {
   detail::stable_set<std::filesystem::path> result;
   const auto bare_mode = caf::get_or(cfg, "vast.bare-mode", false);
-  if (!bare_mode) {
+  if (auto dirs = caf::get_if<std::vector<std::string>>( //
+        &cfg, "vast.plugin-dirs"))
+    result.insert(dirs->begin(), dirs->end());
+  if (!bare_mode)
```

Comment:
Can you copy the content of the commit message to a comment above line 57? It looks wrong without the context.